epf/src/main/java/com/daasuu/epf/EPlayerView.java

```diff
@@ -31,11 +39,14 @@ public EPlayerView(Context context, AttributeSet attrs) {
         super(context, attrs);
 
         setEGLContextFactory(new EContextFactory());
-        setEGLConfigChooser(new EConfigChooser());
+
+        setEGLConfigChooser(new EConfigChooser(8, 8, 8, 8, 16, 0, EGL_CONTEXT_CLIENT_VERSION));
+
+        setZOrderOnTop(true);
```

Comment:
It cannot be merged as it will affect other views.

Reply:
Fair point. I've solved the issue by providing a choice as to whether this functionality is desired

Reply:
Fair ??? I dont think fair.

Reply:
Sorry, what?